Add glossary of terms to the `Getting Started` page (#283)
BrownBiomechanics/Autoscoper: Documentation/getting-started.md

* Ankle BVR data - This data was provided by Michael Rainbow.
  * Three frames of movement are included in the sample data.
  * Three partial volumes are included in the sample data. The tibia, talus, and calcaneus are included.

## Glossary

This section defines terms that are commonly used by Autoscoper and SlicerAutoscoperM

> * **3D Computed Tomography (CT)**: Computerized x-ray imaging method producing volumetric density data of a region of interest. Saved in DICOM format.

Reviewer: ```suggestion
* **3D Computed Tomography (CT)**: A computerized X-ray imaging method that produces volumetric density data of a region of interest. The data is typically saved in DICOM format.
```

> * **4D CT**: Temporal 3D CT

Reviewer: ```suggestion
* **4D CT**: Temporal 3D CT.
```

> * **Biplaner Videoradiography (BVR)**: Two calibrated x-ray sources and high-speed video camera sets that allow for the synchronous collection of radiographs from two perspectives. Camera models from the calibrations allow for unified 3D World construction from perspective 2D views.

Reviewer: ```suggestion
* **Biplaner Videoradiography (BVR)**: A system utilizing two calibrated X-ray sources and high-speed video cameras to simultaneously capture radiographs from two perspectives. The camera models derived from the calibrations enable the construction of a unified 3D World from the 2D perspective views.
```

* **Rigid Body**: Commonly a bone or an implant. The rigid body is the region of interest within a 3DCT. Users segment or isolate the data which represents the rigid body. Often the rigid body surface file is exported as a bone mesh model (STL). Autoscoper is used to track the kinematics (rotation and translation) of the rigid body in 3-dimensional space.

> * **Partial Volume (PVOL)**: Saved as a TIFF stack, the PVOL is a volumetric image data file. A PVOL is a subset of 3DCT scan data as defined from a segmented rigid body and cropped to its dimensions.

Reviewer: ```suggestion
* **Partial Volume (PVOL)**: A volumetric image data file saved as a TIFF stack. The PVOL represents a subset of 3D CT scan data, defined by a segmented rigid body and cropped to its dimensions.
```

> * **AUT Model (AUT)**: A representation of the rigid body STL as parsed in the Autoscoper 3D World

Reviewer: ```suggestion
* **AUT Model (AUT)**: A representation of the rigid body STL file, as parsed and visualized within the Autoscoper 3D World.
```

> * **Digitally Reconstructed Radiograph (DRR)**: When a version of the PVOL is loaded into Autoscoper, the perspective camera models cast the volumetric density data onto the same image plane as the radiographs to generate a synthetic radiograph

Reviewer: ```suggestion
* **Digitally Reconstructed Radiograph (DRR)**: A synthetic radiograph generated when a PVOL is loaded into Autoscoper. Perspective camera models cast the volumetric density data onto the same image plane as the radiographs, simulating the radiograph digitally.
```

I removed the words `a version of the` associated with the PVOL. If we can clarify, this can be added back.

> * **Radiograph**: A 2D image produced by x-rays.

Reviewer: ```suggestion
* **Radiograph**: A 2D image produced by X-rays.
```

> * **Sequential 3D CT**: A collection of 3D CT scans acquired of the same region of interest (joint or limb) in different positions. Using the Sequences model, the independent volumes can be visualized and processed together in a series.

Reviewer: ```suggestion
* **Sequential 3D CT**: A series of 3D CT scans acquired from the same region of interest (such as a joint or limb) in different positions. Using the Sequences module, these independent volumes can be visualized and processed together as a series.
```

I am assuming `Sequences module` should be used instead of `Sequences model`

> * **TFM**: An ITK transformation file with extension `.tfm`.

Reviewer: ```suggestion
* **TFM**: An ITK transformation file with the extension `.tfm`.
```

* Ankle BVR data - This data was provided by Michael Rainbow.
  * Three frames of movement are included in the sample data.
  * Three partial volumes are included in the sample data. The tibia, talus, and calcaneus are included.

## Transformation Overview

This section will contain a figure that shows the different coordinate systems used and the associated transformations between them.

## Glossary

This section defines terms that are commonly used by Autoscoper and SlicerAutoscoperM

> * **3D Computed Tomography (CT)**: A 3D volume obtained from x-rays.

Reviewer 2: I'm assuming there are more concrete definitions for 3DCT , 4DCT than these ?

Reviewer 2: - ensure density volumetric data as part of def.

* **4D CT**: Like 3D CT but with added motion over time.
* **Biplaner Videoradiography (BVR)**: Two calibrated x-ray sources that allow for the simultaneous collection of radiographs from two perspectives.
* **Digitally Reconstructed Radiograph (DRR)**: Synthetic radiograph generated from 3D CT using real world camera parameters.

> * **Partial Volume (PVOL)**: An individual rigid body cropped from a CT scan.

Reviewer 2: suggest moving PVOL definition up above DRR

Reviewer 2: PVOL: Saved as a TIFF stack, the PVOL is a volumetric image data file.  A PVOL is a subset of 3DCT scan data as defined from a segmented rigid body, and cropped to its dimensions.

* Ankle BVR data - This data was provided by Michael Rainbow.
  * Three frames of movement are included in the sample data.
  * Three partial volumes are included in the sample data. The tibia, talus, and calcaneus are included.

## Transformation Overview

> This section will contain a figure that shows the different coordinate systems used and the associated transformations between them.

Reviewer: This should probably be added later

* **Rigid Body**: Commonly a bone or an implant. The rigid body is the region of interest within a 3DCT. Users segment or isolate the data which represents the rigid body. Often the rigid body surface file is exported as a bone mesh model (STL). Autoscoper is used to track the kinematics (rotation and translation) of the rigid body in 3-dimensional space.
* **Partial Volume (PVOL)**: Saved as a TIFF stack, the PVOL is a volumetric image data file. A PVOL is a subset of 3DCT scan data as defined from a segmented rigid body and cropped to its dimensions.
* **AUT Model (AUT)**: A representation of the rigid body STL as parsed in the Autoscoper 3D World
* **Digitally Reconstructed Radiograph (DRR)**: When a version of the PVOL is loaded into Autoscoper, the perspective camera models cast the volumetric density data onto the same image plane as the radiographs to generate a synthetic radiograph
* **Radiograph**: A 2D image produced by x-rays.
* **Sequential 3D CT**: A collection of 3D CT scans acquired of the same region of interest (joint or limb) in different positions. Using the Sequences model, the independent volumes can be visualized and processed together in a series.

> * **TFM**: An ITK transformation file.

Reviewer: ```suggestion
* **TFM**: An ITK transformation file with extension `.tfm`.
```

> * **TRA**: A file extension exported by Autoscoper, containing Nx16 transformation matrix of rigid body kinematics. N line numbers is equal to the number of frames of BVR data optimized in the Autoscoper project TRA transforms can be applied to AUT models to visualized 3D motion of the rigid body and calculate motion.

Reviewer: ```suggestion
* **TRA**: A transform file with extension `.tra` exported by Autoscoper. It contains a Nx16 transformation matrix of rigid body kinematics. Each one of the N line represent a  frame of BVR data optimized by Autoscoper. The TRA transforms can be applied to AUT models to visualized 3D motion of the rigid body and calculate motion.
```
